agent/util-scripts/pbench-tool-meister-start

```diff
@@ -716,6 +729,7 @@ def main(_prog: str, cli_params: Namespace) -> int:
             logger.error("required ssh command not in our PATH")
             return ReturnCode.MISSINGSSHCMD
     else:
+        assert cli_params.orchestrate == "existing"
```

Comment:
Single-parameter `assert`'s report an `AssertionError` with no other information.  I recommend using the two-parameter version, supplying a useful message (including the value of `cli_params.orchestrate`, hopefully delimited in quotes or otherwise set up to show embedded blanks).